.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.ts.snap linguist-generated=true
```

Comment:
collapses the .snap file in diffs

Reply:
is this in the terminal where it's collapsed?

Reply:
![image](https://github.com/guardian/support-dotcom-components/assets/114918544/d8532418-03a7-405a-afb4-a97383a7c056)
I normally see this on GitHub so thought this was for somewhere else